---
title: 'User Manual: PBR_32803 QTL mapping' 
author: "Dennis van Muijen & David Nieuwenhuijse"
output:  
  pdf_document:
    number_sections: false
---

**Important note: The shiny app disconnects after 30 minutes of no activity after which your data is lost. The app is hosted on a cloud server for which limited hours of running time is available.**

# Uploading data
The main tab "Upload data" is used to load a cross file using the "Browse" button.
The summary panel gives a summary of the cross file, including the number of samples, markers, and chomosomes. The number of selfings can be selected from the menu: F2 for the simulated data and F6 for the RIL population. The change is adapted immediately and can  be selected before or after uploading the data. Make sure .csv files are used and not the files with a .geg extension when analysing the F2 simulated data. 

# Interval mapping
## Input summary
A summary is printed giving statistics of the uploaded cross file such as number of individuals, markers, and phenotypes. Also the selected number of selfings is printed. After uploading a cross file two additional sidebar menu's are activated: one to select the phenotype column to analyse and a menu to select the chromosomes to analyse. By default the first phenotype and all chromosomes are selected. To change a phenotype, select one from the list. To change the chromosomes to analyze, select the ones you would like to remove and press backspace or delete. Multiple chromosomes can be selected and deleted by holding control or shift while selecting. The vertical black lines on the x-axis indicate where true markers were mapped. 

## LOD Profile
Interval mapping is run given the selected number of selfings, phenotype, and chromosomes. The calculation can take 1-3 seconds due to the 2000 permutations that are calculated. In the permutation test the phenotypic values are randomly assigned to the different genotypes. Each iteration returns the highest LOD score of a detected false QTL. Significance thresholds are subsequently derived from the permutations. Significance thresholds at alpha=0.01 and alpha=0.05 are added to the figure as dashed orange and white lines respectively. A qtl is considered at the alpha = 0.05 threshold corresponding to a 5% genome-wide error rate.  

## Fit qtl
Given QTL were found with interval mapping, the QTL can be fitted as (multiple) QTL model to derive Type III ANOVA LOD scores for the individual QTL and the LOD score of the full QTL model, *e.g* y ~ QTL1 + QTL2 + QTL3. Additionally, additive and dominance effects are printed under estimated effects. For example, 8@56.3a  corresponds the additive effect found on chromosome 8 at 56.3 cM 8@56.3d to the dominance effect. Besides the estimates, standard errors and t-values are printed in additional columns.

# Composite Interval mapping

## LOD Profile
LOD scores per marker/pseudo-marker is shown on the y-axis and the cM positions on the x-axis. Both the LOD profile for Interval Mapping is shown (blue) as well the LOD profile for composite interval mapping (red). Composite interval mapping provides additional power to detect QTL by controlling background noise using marker co-factors. There are different approaches to select the number of cofactors; a common approach is to select the number of QTL detected in Interval Mapping as the number of co-factors **(Malosetti *et al* 2013)**. This strategy is used in the Shiny app. If there are no QTL detected in Interval Mapping, Composite Interval Mapping is carried with one additional co-factor. Note that selecting no cofactors in CIM provides exact the same results as Interval Mapping. 

## Fit qtl
Identical to the menu for Interval Mapping except that now the QTL are fitted that are found using Composite Interval Mapping. 


#About
Contains version number, author information and credits to used R packages.

Reference used: **The statistical analysis of multienvironment data: modeling genotype-by-environment interaction and its genetic basis.** *Marcos Malosetti, Jean-Marcel Ribaut & Fred A. van Eeuwijk 2013*
